### DISCUSSION TITLE
Why no thumbs down emoji for issues/discussion threads?

### DISCUSSION THREAD

**Discussion by DiscoFever78** [community/community#70486] [Other Feature Feedback, Questions, & Ideas]
Just curious :) 

Update:
Edited out wordiness, and the title says it all :)

**Comment by seyLu** [comment]
#### Hi @DiscoFever78,

Wdym?
![image](https://github.com/community/community/assets/98249191/7f2a5b53-4e91-41a8-999b-fcf4789fd89d)
![image](https://github.com/community/community/assets/98249191/1ca394ba-1d2f-4ee3-8921-5cda93bded18)

**Comment by DiscoFever78** [comment]
Strange, it wasn't showing up for me. 

![no thumbs down on gitbub1](https://github.com/community/community/assets/67702582/fdf32a2f-f46e-4193-ba91-5c49c5ca6031)
